EPCPortalWeb/Models/ReportDataModel.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace EPCPortalWeb.Models
+{
+    public class ReportDataModel
```

Comment:
Feel like this could be split into two classes: one is the pure Model, that contains the various cost properties.

The other would be a service that generates the list of recommendations and their savings - this latter class would hold the methods that currently sit in this class.